packages/pyxel/meta.yaml

```diff
@@ -1,6 +1,7 @@
 package:
   name: pyxel
   version: 1.9.10
+  _disabled: true
```

Comment:
What error do we get for pyxel?

Reply:
Build succeeds, but get function sig mismatch or null in several tests.